subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -1806,15 +1877,26 @@ uint16_t isoal_tx_unframed_get_next_payload_number(isoal_source_handle_t source_
 	return sdus_skipped;
 }
 
-/* NOTE: Use of target_event and grp_ref_point as input from upper layer.
+/**
+ * @brief Fragment received SDU and produce unframed PDUs
+ * @details Destination source may have an already partially built PDU
  *
- * For unframed:
+ * @param source_hdl[in] Destination source handle
+ * @param tx_sdu[in]     SDU with packet boundary information
+ *
+ * @return Status
+ *
+ * @note
+ * PSN in SDUs for unframed TX:
+ *
+ * @par
  * Before the modification to use the PSN to decide the position of an SDU in a
  * stream of SDU, the target event was what was used in deciding the event for
  * each SDU. This meant that there would possibly have been skews on the
- * receiver for each SDU and we had trouble with LL/CIS/PER/BV-39-C which
+ * receiver for each SDU and there were problems  with LL/CIS/PER/BV-39-C which
  * expects clustering within an event.
  *
+ * @par
```

Comment:
Yes but this is an extension of the `note` so as I understand it will end on a blank line unless extended by the `par`. I was following the example for multi paragraph [note](https://www.doxygen.nl/manual/commands.html#cmdpar).